docs/install.md

```diff
@@ -4,7 +4,7 @@ Although the controller can be used independently from the aws-app-mesh-inject w
 
     # use `export MESH_NAME=color-mesh` to work with the example in this repository.
     export MESH_NAME=<my-mesh-name> 
-    curl https://raw.githubusercontent.com/aws/aws-app-mesh-inject/v0.1.0/hack/install.sh | bash
+    curl https://raw.githubusercontent.com/aws/aws-app-mesh-inject/v0.1.0/scripts/install.sh | bash
```

Comment:
I couldn't see `install.sh` in that folder.

Reply:
There's separate PR for adding that folder. 

https://github.com/aws/aws-app-mesh-inject/pull/51